lms/static/sass/inline-discussion.scss

```diff
@@ -0,0 +1,22 @@
+// From lms-main-v1.scss
```

Comment:
Same question as from the rtl file: why should all these imports be duplicated? I'd rather see a shared Sass partial that both files can include.